src/Sulu/Component/SmartContent/Orm/DataProviderRepositoryTrait.php

```diff
@@ -339,16 +341,6 @@ private function appendRelationAnd(QueryBuilder $queryBuilder, $relation, $value
         return $parameter;
     }
 
-    /**
-     * Creates a new QueryBuilder instance that is prepopulated for this entity name.
-     *
-     * @param string $alias
-     * @param string|null $indexBy
-     *
-     * @return QueryBuilder
-     */
-    abstract public function createQueryBuilder($alias, $indexBy = null);
```

Comment:
adding the return type here would a major BC Break making lot of bundles fail. So removing the abstract and move it to a phpdoc is only way for Backards compatibility to orm 2 based bundles.